src/encoded/static/components/body_map.js

```diff
@@ -602,24 +613,25 @@ class BodyMap extends React.Component {
         return (
             <div className="body-facet-container">
                 <div className="body-list body-list-top">
-                    <div className="body-list-inner">
+                    <ul className="body-list-inner">
                         {Object.keys(SystemsList).map(b =>
-                            <button
-                                className={`body-list-element ${((typeof this.state.selectedOrgan === 'string' && this.state.selectedOrgan === b) || (typeof this.state.selectedOrgan !== 'string' && this.state.selectedOrgan.includes(b))) ? 'active' : ''}`}
-                                id={b}
-                                onClick={e => this.chooseOrgan(e)}
-                                onMouseEnter={e => highlightOrgan(e)}
-                                onMouseLeave={unHighlightOrgan}
-                                key={b}
-                            >
-                                {b}
-                            </button>
+                            <li key={b}>
+                                <button
+                                    className={`body-list-element ${((typeof this.state.selectedOrgan === 'string' && this.state.selectedOrgan === b) || (typeof this.state.selectedOrgan !== 'string' && this.state.selectedOrgan.includes(b))) ? 'active' : ''}`}
```

Comment:
typeof to me is a code smell in dynamic languages. selectOrgan can be a string or array. Why not always have it be a string ?

Reply:
This is a very good point, and I think this code should definitely be fixed up. `this.state.selectedOrgan` should probably always be an array. I ran into some coding issues when I tried to implement it this way - it's less trivial than it seems like it should be. I'd like to fix it, but I think it may require some work and would prefer to do it as part of another ticket, just because this already has so much going on.